rectangle.py

```diff
@@ -4,4 +4,11 @@ def perimeter(length, breadth):
 
 def area(length, breadth):
   a = length * breadth
-  return a
\ No newline at end of file
+  return a
+
+
+l = int(input('enter the length: '))
```

Comment:
can u please use self explanatory variable names

```suggestion
length = int(input('enter the length: '))
```